tuning/sft_trainer.py

```diff
@@ -607,10 +624,19 @@ def main():
 
     combined_tracker_configs.file_logger_config = file_logger_config
     combined_tracker_configs.aim_config = aim_config
+    sc_callback = None
 
     if training_args.output_dir:
         os.makedirs(training_args.output_dir, exist_ok=True)
         logger.info("using the output directory at %s", training_args.output_dir)
+        if add_scanner_callback:
+            output_fmt = os.path.join(training_args.output_dir, "scanner_output.json")
+            sc_callback = [Scanner(output_fmt=output_fmt)]
```

Comment:
@aluu317 
Can we check again if the package exists here? if exists we add the callback if not we can raise a ValueError or warn the user and fall back not adding the scanner. Either ways is fine. 

Thank you.

Reply:
sounds good. I added another check. sorry for delay, i was OOO for a few days. Please review again